helm/values.yaml

```diff
@@ -0,0 +1,205 @@
+  # The PCCS uses this API key to request collaterals from Intel's Provisioning Certificate Service. User needs 
+  # to subscribe first to obtain an API key. For how to subscribe to Intel Provisioning Certificate Service and
+  # receive an API key, goto https://api.portal.trustedservices.intel.com/provisioning-certification and click
+  # on 'Subscribe'.
+  apiKey: "aecd5ebb682346028d60c36131eb2d92"
+
+  # Specify the proxy server for internet connection, for example, "http://192.168.1.1:80". Leave blank for no proxy or system proxy.
+  proxy: ""
+
+  # cron-style refresh schedule for the PCCS to refresh cached artifacts including CRL/TCB Info/QE Identity/QVE Identity. The default
+  #setting is "0 0 1 * * *", which means refresh at 1:00 AM every day. Check https://docs.gitlab.com/topics/cron/ for more examples
+  refreshSchedule: "0 0 1 * * *"
+
+  # Sha512 hash of the user token for the PCCS client user to register a platform. For example, PCK Cert ID retrieval tool will
+  # use the user token to send platform information to PCCS.
+  userTokenHash: "6ae2d3bfb3b95517b358fcdb29f7743246101ebf13f797d8244df795eec2d1d769a41c059dc37beadac8e40cecab4352764336a90302920ddeb1a6c6df4e8a00"
+
+  # Sha512 hash of the administrator token for the PCCS administrator to perform a manual refresh of cached artifacts.
+  adminTokenHash: "31a556961f3438f9f632ca27812d22228e98e5083eea2bf9b78d0bb374d44deb0dc4d1bc16ab64127eb74e8452ea6902d97937c28310a7ab62a9ae1c15d96d69"
+
+  # Caching mode options: LAZY, OFFLINE, or REQ
+  # https://github.com/intel/SGXDataCenterAttestationPrimitives/tree/dcap_1.21_reproducible/QuoteGeneration/pccs#caching-fill-mode
+  cachingFillMode: "LAZY"
+
+  # OpenSSL FIPS mode (set `true` to enforce FIPS 140-2 compliance)
+  opensslFipsMode: false
+
+  # Log level. Use the same levels as npm: error, warn, info, http, verbose, debug, silly. Default is info.
+  logLevel: "debug"
+
+  storage:
+    # Database type: use "sqlite" or "mysql"
+    dialect: "sqlite"
+
+    # Database connection settings
+    database: "pckcache"
+    username: "username"
+    password: "password"
+    host: "localhost"
+
+    # MySQL-specific settings (ignored for SQLite)
+    port: 3306
+
+    # Connection pool configuration
+    pool:
+      max: 5          # Maximum number of connections
+      min: 0          # Minimum number of connections
+      acquire: 30000  # Max time (ms) to try acquiring a connection before throwing an error
+      idle: 10000     # Max time (ms) a connection can remain idle before being released
+
+    # Ensure Sequelize uses table names as defined in models
+    freezeTableName: true
+
+    # Enable or disable SQL query logging (set to `true` for debugging)
+    logging: false
+
+    # SQLite storage file (ignored for MySQL)
+    filename: "pckcache.db"
+
+    # MySQL SSL settings (only required if connecting over SSL)
+    ssl:
+      required: true
+      ca: "/if_required/path/to/your_ssl_ca"
+
+# PCCS TLS key configuration
+pccsKeys:
```

Comment:
should it be a tls key pair? If the field is named clearly, the comment here is not necessary.